test/e2e-tests/tests/sanity_test.go

```diff
@@ -1,19 +1,68 @@
 package tests
 
 import (
-	"context"
+	"github.com/opendatahub-io/ai-edge/test/e2e-tests/oc"
 	support "github.com/opendatahub-io/ai-edge/test/e2e-tests/support"
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"testing"
 )
 
-func TestGetAllPods(t *testing.T) {
+const (
+	ContainerImagePipeline    = "../../../pipelines/tekton/build-container-image-pipeline/"
+	ContainerImagePipelineRun = "../../../pipelines/tekton/build-container-image-pipeline/build-container-image-pipelinerun-bike-rentals.yaml"
+)
+
+func Test_ConnectingToCluster(t *testing.T) {
+	_, err := support.ClusterClientSet()
+	if err != nil {
+		t.Fatal(err.Error())
+	}
+}
+
+func Test_ContainerImagePipelineKustomizeApply(t *testing.T) {
+	clientSet, err := support.ClusterClientSet()
+	if err != nil {
+		t.Fatal(err.Error())
+	}
+
+	ctx := support.CreateContext()
+
+	if false {
+		support.CreateTestNamespace(ctx, clientSet)
+		defer support.DeleteTestNamespace(ctx, clientSet)
+	}
```

Comment:
While these create and delete the namespace as expected. Using `oc. project` to set them as the current namespace for all other oc commands does not work. Whatever project you were using before testing will be used.

Deleting the namespace does not wait for it to be deleted. Which means calling multiple tests with this will cause an error because you are trying to create and delete the namespace at the same time